src/OrchardCore/OrchardCore.Users.Core/Services/UserService.cs

```diff
@@ -12,13 +12,13 @@ namespace OrchardCore.Users.Services
     /// <summary>
     /// Implements <see cref="IUserService"/> by using the ASP.NET Core Identity packages.
     /// </summary>
-    public class UserService : IUserService
+    public class UserService : UserClaimsPrincipalFactory<IUser>, IUserService
```

Comment:
Why here and not the `UserManager` ?

Reply:
I wanted to use the default implementation of UserClaimsPrincipalFactory<TUser> and it depends on UserManager<TUser> and IOptions<IdentityOptions> , so I selected UserService which has the same dependencies.